### ISSUE TITLE
Create Issues for 'Road to Glory Mode' 

### ISSUE THREAD

**Issue by antma21n** [antma21n/UnityCFBGame#12]
Once fundamental gameplay is complete the road to glory mode needs to be added. Start by writing tickets.

Systems Notes:

1. Team Hub: main area where player gets updates from other systems and can start next game with a game plan
	- Start next game
	- Quick Team Stats
	- Quick Recuiting Stats
	- CFB rankings
	- Opponent Info
	- GamePlan 
2. Manage Team: manage players, boosters, AD and fans
	- Player Data
		- Rating
		- Game Stats
		- Morale
		- Health
		- Traits 
	- Boosters
	- AD Management 
	- Fan Managerment 
3. [Important but not priority] Recuiting
	- Recruit Generator
	- Recruit List
		- all recruitable players
	- Recruit Stats
		- Scouted, Potential Rating, Actual Rating, Position, Commitment Level
	- Watch List
		- players you want
	- Recruit Actions: Spend recruit points to sign players
		- Visits
		- Coach Visit
		- Scout
		- NCAA Game Visit
		- HS Game Visit
		- Workout 
		- Scholarship Offer 
		- Sign Player
4. Around the League: A screen to display a simulated NCAA to see how your rivals and other teams are doing
	- CFB Polls
	- Team Ratings
	- Team Game Stats
	- Simulation week to week of games 
	- Start of Season Schedule Creator
5. [NICE TO HAVE] Stats and HOF: A screen to display NCAA record and your favorite players to play for you
	- Record Screen
	- HOF screen
6. [NICE TO HAVE] Options
	- various settings
	- sound settings
	- quit game

**Comment by antma21n** [comment]
First goal of road to glory should be to **play 12 games, and a bowl game with a realistic schedule**. Each team should have variable difficulty from their rating. Should have ability to see around CFB games. 

May need to create a new project/epic for this
